scripts/publish_bumpver_handler.py

```diff
@@ -0,0 +1,95 @@
+# this file is intended to be called by a github workflow (.github/workflows/publish_to_pypi.yaml)
+# it makes decisions based on the current version and the component specified for bumping,
+# which the workflow cannot do
+
+"""
+TESTING:
+- add and commit any changes (keep track of this commit hash)
+- bumpver update --set-version 1.0.0
+
+- python publish_bumpver_handler.py
+  - expect: ValueError
+
+- python publish_bumpver_handler.py fake
+  - expect: ValueError
+
+- python publish_bumpver_handler.py major
+  - expect: version updated to 2.0.0
+
+- python publish_bumpver_handler.py minor
+  - expect: version updated to 2.1.0
+
+- python publish_bumpver_handler.py patch
+  - expect: version updated to 2.1.1
+
+- python publish_bumpver_handler.py dev
+  - expect: version updated to 2.1.2.dev0
+
+- python publish_bumpver_handler.py dev
+  - expect: version updated to 2.1.2.dev1
+
+- python publish_bumpver_handler.py major
+  - expect: ValueError
+
+- python publish_bumpver_handler.py minor
+  - expect: ValueError
+
+- python publish_bumpver_handler.py patch
+  - expect: version updated to 2.1.2
+
+- git reset --hard {hash of the commit made at the beginning}
+- git tag --delete 1.0.0 2.0.0 2.1.0 2.1.1 2.1.2 2.1.2.dev0 2.1.2.dev1
+"""
+
+import subprocess  # nosec
```

Comment:
What does `#nosec` do?

Reply:
bandit (part of our precommit) flags possible security vulnerabilities and it was mad about subprocess use, nosec skips that line